src/win/conpty.cc

```diff
@@ -8,6 +8,12 @@
  *   with pseudo-terminal file descriptors.
  */
 
+// node versions lower than 10 define this as 0x502 which disables many of the definitions needed to compile
+#include <node_version.h>
```

Comment:
@deepak1556 will this work in NAPI?

This header is also used in pty.cc: https://github.com/microsoft/node-pty/blob/4043efdc49c862ef9be037cf22ba15bec305474e/src/unix/pty.cc#L41-L45, but we should rip it out as we don't even support that version anymore.

Reply:
Yup its node header that can be used alongside napi.